.gitignore

```diff
@@ -24,6 +24,5 @@ yarn-debug.log*
 yarn-error.log*
 
 #credentials
-/.env
```

Comment:
it's necessary to leave it here to prevent downloading mentioned file.

Reply:
Slash from start means that  file is global. So that's not necessary =)

Reply:
Resolved